### PR TITLE
docs: adds code coverage reporter config example

### LINKED ISSUE #1548: [test-runner] Implement cobertura report
Hello,

Since my company use azure devops, we can get coverage report from build pipeline via cobertura reports.

https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/publish-code-coverage-results?view=azure-devops

>For example: $(System.DefaultWorkingDirectory)/MyApp/**/site/cobertura/coverage.xml

I tried to look in web/test-runner code or documentation but I didnt' found anything related to that.

How should I do to get thoses reports ? Is it possible ?

### PR DESCRIPTION
**Pull request by gavinbarron**
adds documentation to show how cobertura can be added to the coverageConfig for enhanced reporting

Closes #1548

### CODE REVIEW

File: docs/docs/test-runner/writing-tests/code-coverage.md [hunk 1]
```diff
@@ -94,3 +94,27 @@ export default {
...
+## Coverage reporting
+
+By default coverage reporting uses the lcov reporter. Should you want to use additional reporters, for example, cobertura, then the `reporter` config element should be modified.
+
+**Example config:**
+
+```js
+// web-test-runner.config.mjs
+
+module.exports = {
```
Comment (koddsson) [inline docs/docs/test-runner/writing-tests/code-coverage.md:107]: This needs to be a module file and not commonJS if it's `.mjs`.

```suggestion
export default {
```

**Review by koddsson — COMMENTED** [review]
A minor change but otherwise this looks good :)

**Comment by gavinbarron** [comment]
@koddsson I've also pulled that change up to the top of the page to the code snippet I copy/pasted to put this change together.

**Review by koddsson — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**